### ISSUE TITLE
[Bug]: AE2 Middle clicks not working

### ISSUE THREAD

**Issue by R4VANG3R** [FTBTeam/FTB-Modpack-Issues#1099]
### Modpack

Direwolf20 1.18

### Modpack version

1.7.0

### Describe the bug

When middle clicking in the AE terminal to craft an item nothing happens, nor does middle click to set the desired amount for crafting recipes work.

### Steps to reproduce

1. Open AE terminal
2. Middle click on a craftable item that also has existing items
3. Nothing happens

or

1. Open ME Pattern Encoding Terminal
2. Add item
3. Middle click to set amount
4. Nothing happens

### Expected behaviour

AE2 middle click functionality should work

**Comment by quietust** [comment]
The problem is that the Inventory Sorter mod's "middle-click" hotkey conflicts with AE2 - if you rebind it to something else, you'll be able to request autocrafting again.

**Comment by R4VANG3R** [comment]
I just tried unbinding everything that was bound to middle mouse, but it didn't resolve the issue

**Comment by mitzee4** [comment]
I believe you have to leave Minecraft's default Pick Block control bound to use AE2's autocrafting. I have everything except that control unbound from middle mouse and am able to utilise middle click for AE2 autocrafting.

**Comment by jkerr** [comment]
I also have this issue in Plexiglass Mountain.  I've also tried unbinding the current two bindings, Sophisticated Backpacks and Inventory Sorter, but that didn't help...  Mentioned on the ae2 discord and got the reply "they may do weird stuff that skips our hooks".

**Comment by Raidobw2** [comment]
Apparently that works by doing the config, which is client sided

https://github.com/cpw/inventorysorter/pull/141

But last time I checked 1.18.2 in-dev, there wasn't a client config for the mod, and putting `"ae2:craftingterm"` in the server config didn't work either.

Maybe we can do some KubeJS magic if InventorySorter's classes are public and not too complex. Kube needs `invertClassLoader=true` in `kubejs/config/common.properties`

I'll look this up for PGM when I've got time and if I succeed, I'll post a solution here.
